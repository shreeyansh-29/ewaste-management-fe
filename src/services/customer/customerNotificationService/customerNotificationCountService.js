import {CUSTOMER_NOTIFICATION_URL} from "../../../container/constant/constant";
import api from "../../../core/utilities/httpProvider";

export const customerNotificationCountService = () => {
  return api.get(CUSTOMER_NOTIFICATION_URL);
};
