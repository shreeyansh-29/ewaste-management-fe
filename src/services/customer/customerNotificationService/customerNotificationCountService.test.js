/* eslint-disable no-undef */

import {customerNotificationCountService} from "./customerNotificationCountService";
import React from "react";
import Enzyme, {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

Enzyme.configure({adapter: new Adapter()});

it("testing customerNotificationCountService", () => {
  const Service = customerNotificationCountService;
  const wrapper = shallow(<Service />);
  expect.assertions(1);
  expect(toJson(wrapper)).toMatchSnapshot();
});
