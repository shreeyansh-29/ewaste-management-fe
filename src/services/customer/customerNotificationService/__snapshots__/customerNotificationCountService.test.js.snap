// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing customerNotificationCountService 1`] = `<Component />`;
